docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>comparison with the one without mixer (plain QAOA)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Maybe include both lines (X and XY) in the same plot so the reader can see the comparison?</p><p> </p>